backend/src/utilities/feature-extractor.js

```diff
@@ -0,0 +1,143 @@
+const fs = require('fs');
+const Papa = require('papaparse');
+const { Product, ProductFeatures } = require('../database/models');
+
+const DEFAULT_CSV_PRODUCT_MAPPING = {
+  ProductRef: 'productId',
+  ProductName: 'name',
+  'Category Name': 'category',
+  'Sub Category': 'subcategory'
+};
+const FEATURE_REGEX = new RegExp('\\w+', 'g');
```

Comment:
I think you deleted the old feature regex here

Reply:
Intentional. Changed the logic of extracting features. Instead of choosing what to split on, I choose what to select ("words"). It works about the same